libraries/ghc-prim/GHC/Types.hs

```diff
@@ -205,13 +205,6 @@ or the 'Prelude.>>' and 'Prelude.>>=' operations from the 'Prelude.Monad'
 class.
 -}
 newtype IO a = IO (State# RealWorld -> (# State# RealWorld, a #))
-type role IO representational
```

Comment:
Is it a good idea to remove this? Was the role just to catch some potential role changes which would require changing `TcForeign` (in which case, yeah, remove).

Reply:
Yes, I believe the comment below was the only reason for the role declaration.